src/player_builder.py

```diff
@@ -8,12 +10,25 @@ def __init__(self, validator, processor, cli):
         self.cli = cli
 
     def build_players(self, player_tokens):
-        human_player_first = HumanPlayer(self.cli, self.validator, token=player_tokens[0])
-        human_player_second = HumanPlayer(self.cli, self.validator, token=player_tokens[1])
-        computer_first_player = ComputerPlayer(self.processor, self.cli, player_tokens[0])
-        computer_second_player = ComputerPlayer(self.processor, self.cli, player_tokens[1])
-        #return [human_player_first, human_player_second]
-        #return [computer_first_player, computer_second_player]
-        return [human_player_first, computer_second_player] 
-        #return [computer_first_player, human_player_second]
+        choice = self.validator.get_valid_player_choice(self.cli)
+
+        player_1 = self.build_human_player(player_tokens[0])
+        if choice == "1":
+            player_2 = self.build_human_player(player_tokens[1])
+        elif choice == "2":
+            player_2 = self.build_easy_computer()
+        elif choice == "3":
+            player_2 = self.build_hard_computer()
+        else:
+            return constants.EXIT
+        return [player_1, player_2]
```

Comment:
I think this method could use a little work, and that may help resolve the issue in my other comment about the implementation of this method's behavior.
https://arstechnica.com/information-technology/2014/12/in-a-dynamically-typed-language-is-it-a-bad-idea-to-return-different-data-types/
https://stackoverflow.com/questions/1839289/why-should-functions-always-return-the-same-type